tests/filecheck/transforms/cse.mlir

```diff
@@ -138,7 +130,7 @@ func.func @different_ops() -> (i32, i32) {
 // We do not have this Region Kind disctinction; so everything here works on the pessimistic
 // Graph Rewgion assumption.
 
-// CHECK-LABEL: @down_propagate
+// CHECK-LABEL: @down_propagate()
```

Comment:
`@down_propagate` was not a unique label, (FileCheck requires labels to be globally unique), so I made a change here to make it unique.